readme.md

```diff
@@ -0,0 +1,243 @@
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`newlineToBreak(tree)`](#newlinetobreaktree)
+*   [Syntax](#syntax)
+*   [Syntax tree](#syntax-tree)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a utility that takes an [mdast][] tree and turns soft line
+endings (enters) into hard breaks (`<br>`s)
+
+This package is used inside [remark-breaks][], which focusses on making it
+easier to transform content by abstracting these internals away.
+
+## When should I use this?
+
+This plugin is useful if you want to display user content closer to how it was
+authored, because when a user includes a line ending, it’ll show as such.
+GitHub does this in a few places (comments, issues, PRs, and releases), but it’s
+not semantic according to HTML and not compliant to markdown.
+Markdown already has two ways to include hard breaks, namely trailing spaces and
+escapes (note that `␠` represents a normal space):
+
+```markdown
+lorem␠␠
+ipsum
+
+lorem\
+ipsum
+```
+
+Both will turn into `<br>`s.
+If you control who authors content or can document how markdown works, it’s
+recommended to use escapes instead.
+
+## Install
+
+This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
+In Node.js (version 14.14+, or 16.0+), install with [npm][]:
+
+```sh
+npm install mdast-util-newline-to-break
+```
+
+In Deno with [`esm.sh`][esmsh]:
+
+```js
+import newlineToBreak from 'https://esm.sh/mdast-util-newline-to-break@3'
+```
+
+In browsers with [`esm.sh`][esmsh]:
+
+```html
+<script type="module">
+  import newlineToBreak from 'https://esm.sh/mdast-util-newline-to-break@3?bundle'
+</script>
+```
+
+## Use
+
+Say we have the following file, `example.md` (note: there are no spaces after
+`a`):
+
+```markdown
+This is a
+paragraph.
+```
+
+And our module, `example.js`, looks as follows:
+
+```js
+import fs from 'node:fs/promises'
+import {fromMarkdown} from 'mdast-util-from-markdown'
+import {newlineToBreak} from 'mdast-util-newline-to-break'
+import {toMarkdown} from 'mdast-util-to-markdown'
+
+const doc = await fs.readFile('example.md')
+const tree = fromMarkdown(doc)
+newlineToBreak(tree)
+
+console.log(toMarkdown(tree))
+```
+
+Now, running `node example` yields:
+
+```md
+This is a\
+paragraph.
+```
+
+## API
+
+This package exports the identifier `newlineToBreak`.
+There is no default export.
+
+### `newlineToBreak(tree)`
+
+Turn normal line endings into hard breaks.
+
+#### Parameters
+
+*   `tree` ([`Content`][content] | [`Root`][root]) — The [mdast][] tree in
+    which to turn normal line endings into hard breaks.
+
+## Syntax
+
+This utility looks for markdown line endings (`\r`, `\n`, and `\r\n`) preceded
+by zero or more spaces and tabs.
```

Comment:
```suggestion
This utility looks for markdown line endings (`\r`, `\n`, and `\r\n`).
```

The spaces/tabs part is no longer true (and shouldn‘t have been needed at least since micromark was introduced)